nodl_to_policy/_common/_profile.py

```diff
@@ -22,7 +22,7 @@
 
 
 # TODO(aprotyas): For now, only use `node.xml`, since NoDL does not describe Lifecycle nodes
-def common_profile() -> etree.ElementTree:
+def common_profile() -> etree._ElementTree:
```

Comment:
Return type changed to `etree._ElementTree` for uniformity throughout source code. Afterall, `etree.ElementTree` is simply a wrapper class around the former.